resolved-concerns/2019-10-17-tbd-on-stable.md

```diff
@@ -0,0 +1,44 @@
+# TBD behavior on stable
+
+## Three categories of behavior
+
+* Undefined Behavior -- always a bug. Conceptually, the "miri virtual machine" panics here. This is the same definition [in use by the unsafe code guidelines](https://rust-lang.github.io/unsafe-code-guidelines/glossary.html#undefined-behavior) group
+* Unspecified Behavior or Details -- the "miri virtual machine" does not panic, but we don't say what happens. As you point out, there are a number of *reasons* we might not have said what happens yet:
+    * We do not expect to *ever* specify this, because we want the freedom to change it.
+        * Example: Rust struct layout (without a repr attribute) -- not strictly a *behavior*, of course
+        * Example: whether some local variable is materialized on the stack.
+    * We *may* specify this at some point in the future, but there are no plans to do so.
+        * Example: Rust ABI compatibility
+        * Example: What symbols get exported by a DLL
+    * We wish to specify this behavior in the near future, as part of the FFI-unwind project
```

Comment:
Bullet-point hierarchy doesn't match the heading, "three categories"... I also feel that most of this section feels redundant with the terminology page.